azure-pipelines-trafficmanager.yaml

```diff
@@ -0,0 +1,118 @@
+name: SDS Azure Platform - Traffic Manager
```

Comment:
why do you need your own pipeline?

if you do need it then you need to fiddle with files_list so the right PRs trigger on each PR but i wouldn't have thought you would need it

Reply:
Removed Traffic Manage pipeline and added to base pipeline.